### PR TITLE
Add node.js electron example for DeepLab

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by Honry** [comment]
@fujunwei, @mingmingtasd, PTAL, thanks!

**Comment by mingmingtasd** [comment]
> Remove useless `package-lock.json` files BTW.

Should we remove package-lock.json here? @fujunwei What's you opinion?

**Comment by Honry** [comment]
>Remove useless package-lock.json files BTW.

>Should we remove package-lock.json here? @fujunwei What's you opinion?

@fujunwei, @mingmingtasd, [`package-lock.json`](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json) could be automatically generated by `npm install`, I don't think we need to maintain this file. :)

**Comment by fujunwei** [comment]
The file is suggested to be committed into source repositories seeing https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json.

**Comment by Honry** [comment]
> The file is suggested to be committed into source repositories seeing https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json.

Oops, I missed this message, thanks @fujunwei! I've added back the `package-lock.json` files.

**Review by fujunwei — APPROVED** [review]
LGTM.

File: node/examples/electron/semantic_segmentation/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# An Electron.js example of Image Classification using webnn-native
```
Comment (huningxin) [inline node/examples/electron/semantic_segmentation/README.md:1]: Semantic Segmentation?

Comment (Honry) [reply node/examples/electron/semantic_segmentation/README.md]: Thanks, fixed.

**Review by huningxin — APPROVED** [review]

**Pull request merged**